utilities/network.py

```diff
@@ -498,7 +498,7 @@ def _get_interface_ips():
         utilities.virt.wait_for_vm_interfaces(vmi=vmi)
         return _extract_interface_ips()
 
-    sampler = TimeoutSampler(wait_timeout=TIMEOUT_2MIN, sleep=1, func=_get_interface_ips)
+    sampler = TimeoutSampler(wait_timeout=TIMEOUT_4MIN, sleep=1, func=_get_interface_ips)
```

Comment:
Adding 2 minutes to tests  (on failure) is a lot.
Can you please run the failed tests numerous times (15-20) to see that this solves the problem?